build.gradle

```diff
@@ -7,7 +7,7 @@ application {
     mainClassName = 'com.heatheryou.httpserver.Main'
 }
 
-version '1.0-SNAPSHOT'
+version '1.0'
```

Comment:
I don't remember exactly how this works, but I vaguely remember you may want the `SNAPSHOT` bit on `master`, and then change it when you cut a release.  Or something like that.  Do you know how this works?

Reply:
Yes, I think what you said is exactly right. `SNAPSHOT` is used to indicate that the application is still being developed. I should add it back 😅